src/app/public/modules/date-pipe/fixtures/date-pipe.component.fixture.ts

```diff
@@ -11,7 +11,7 @@ import {
   templateUrl: './date-pipe.component.fixture.html'
 })
 export class DatePipeTestComponent {
-  public dateValue: any = new Date('01/01/2000');
+  public dateValue: any = new Date(2000, 0, 1);
```

Comment:
Changing the value in the fixture makes me nervous... Would keeping it as a string make the tests fail? If so, would these changes affect anything the consumer is already doing?

Reply:
Keeping it as a string was making me nervous 😆 

We already know browsers interpret strings passed into the date constructor differently, so this could potentially pass on one browser, but fail on another. We've been seeing builds randomly fail with these date tests lately, so this was one easy change to help mitigate that as much as possible. We should also recommend to our users that constructing new dates with strings is bad practice.

Reply:
So, just to confirm, the changes in this pull request won't affect existing users that use string dates?

Reply:
Only in a good way. My change should improve users using string dates b/c `moment` will now be handling it. For example, `2020-03-03T00:00:00` used to be parsed as UTC on Safari, but after this change it will correctly parse it as local time (thanks to `moment`).

This fixture didn't need to change because of any logic changes. It was just an attempt to fix a bad practice.

Reply:
Would it feel better if I had a test cover string-constructed dates as well? (like it did before)